biogeochem/FatesAllometryMod.F90

```diff
@@ -541,6 +547,11 @@ subroutine carea_allom(dbh,nplant,site_spread,ipft,crowndamage,c_area,inverse)
           call carea_3pwr(dbh_eff,height,ipft,dbh_maxh, site_spread,d2bl_p2, &
                d2bl_ediff, d2ca_min,d2ca_max,crowndamage, c_area, do_inverse)
           capped_allom = .true.
+       case(5)
+          dbh_eff = min(dbh,dbh_maxh)
+          call carea_2pwr(dbh_eff,site_spread,d2bl_p2,d2bl_ediff,d2ca_min,d2ca_max, &
+               crowndamage, c_area, do_inverse)
+          capped_allom = .true.
```

Comment:
I suggest removing `case (5)` and replacing `case (3)` with `case(3,5)`. This way future changes to `care_2pwr` don't need to be applied twice.

Reply:
Thanks Marcos! I have made changes in response to you comments on derivative calculation and the crown area function. But I haven't changed the agb and bleaf allometry to generic function yet. Hope now the code looks less redundant.

Reply:
@XiulinGao Thanks for the follow up. My comment about the generic function was something for a future pull request, not yours. This would likely require significant changes in the parameter files for most users, so it's probably something we want to discuss with the broader FATES community before implementing.